src/net_processing.cpp

```diff
@@ -4386,77 +4460,9 @@ void PeerManagerImpl::ProcessMessage(CNode& pfrom, const std::string& msg_type,
 
     if (msg_type == NetMsgType::BLOCKTXN)
     {
-        // Ignore blocktxn received while importing
```

Comment:
It would be good to explain whether you are modifying the behavior or if this is a refactor. Currently your change forces nodes to de serialize `BLOCKTXN` messages fully before throwing them away. Previously the message was thrown away early.

Reply:
Done it in last force-pushed.